src/Voron/Data/Tables/TableSchema.IndexDefinitions.cs

```diff
@@ -324,22 +326,45 @@ public byte[] Serialize()
                 }
             }
 
+            private const int SupportDuplicateKeysSmallTableIndex = 6;
+            private enum DynamicIndexTable
+            {
+                Type,
```

Comment:
it is a good practice, when value cannot change to not rely on order, but to specify the value explicitly e.g. Type = 0,
IsGlobal = 1

Reply:
done